CategoryConstructor/tst/100_LoadPackage.tst

```diff
@@ -11,13 +11,17 @@ gap> LoadPackage( "Locales", false );
 true
 gap> LoadPackage( "SubcategoriesForCAP", false );
 true
+gap> LoadPackage( "FreydCategoriesForCAP", false );
```

Comment:
Just to be sure, this needs to be generated by an update to `PackageJanitor`.

Reply:
Yes, I will add it to package janitor

Reply:
Thanks!